contribs/github-bot/internal/config/config.go

```diff
@@ -55,10 +56,14 @@ func Config(gh *client.GitHub) ([]AutomaticCheck, []ManualCheck) {
 			Then:        r.Never(),
 		},
 		{
-			Description: "Pending initial approval by a review team member (and label matches review triage state)",
+			Description: "Pending initial approval by a review team member, or review from tech-staff",
 			If:          c.Not(c.AuthorInTeam(gh, "tech-staff")),
 			Then: r.
-				If(r.Or(r.ReviewByOrgMembers(gh, 1), r.Draft())).
+				If(r.Or(
+					r.ApprovalByOrgMembers(gh, 1),
+					r.ReviewByTeamMembers(gh, "tech-staff"),
```

Comment:
Nitpick: I think it's a bit confusing that in some cases (user, org member) the requirement is necessarily an approval, and in other cases (team member), we can specify a review with different possible states. 

If I had to choose, I would have found it much clearer to keep all the requirements in the form of a ⁠`ReviewBy` with your addition of `⁠WithDesiredState()` or a parameter in the constructors.

Reply:
OK; I'll do the changes on all other ones :)

Reply:
done